src/components/BookList.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Book from './Book';
+
+const mapStateToProps = state => ({ books: state.books });
+
+const ConnectedBookList = ({ books }) => (
+  <>
+    <table>
+      {
+        books.map(item => (
+          <tbody key={item.title}>
+            <Book books={item} />
+          </tbody>
+        ))
+      }
+    </table>
+  </>
+);
+
+ConnectedBookList.propTypes = {
+  books: PropTypes.objectOf(PropTypes.string).isRequired,
+};
+
+const BookList = connect(mapStateToProps)(ConnectedBookList);
+
+export default BookList;
```

Comment:
- [ ] [REQUIRED] As stated in the earlier review, this BookList component should be moved to the containers folder. This is because it is not presentational (i.e. rendering only HTML) but is also involved in data fetching.

Reply:
Apologies forgot this file, but I still didn't understand completely why different folders.